utils/docker-ce/files/etc/config/dockerd

```diff
@@ -1,11 +1,18 @@
+# The following settings require a restart to take full effect, A reload will
+# only have partial or no effect:
+# option bip
+# list blocked_interfaces
 
 config globals 'globals'
 #	option alt_config_file "/etc/docker/daemon.json"
 	option data_root "/opt/docker/"
 	option log_level "warn"
 	list hosts "unix:///var/run/docker.sock"
-	# If the bip option is changed, dockerd must be restarted.
-	# A service reload is not enough.
 	option bip "172.18.0.1/24"
 #	list registry_mirrors "https://<my-docker-mirror-host>"
 #	list registry_mirrors "https://hub.docker.com"
+
+# Docker ignores fw3 rules and by default all external source IPs are allowed
+# to connect to the Docker host. See https://docs.docker.com/network/iptables/
+config firewall 'firewall'
+	list blocked_interfaces 'wan'
```

Comment:
@G-M0N3Y-2503 Would it not be better if we move the list blocked_interfaces to the globals section?
And from a security point of view would it not be better if we did not block everything and only allow those who are on the list. Then it is not possible to make all containers public by accident when a new interface are added

Reply:
I think this option isn't documented well enough right now, the initial intention was to have it relate to [the docker documentation on this subject](https://docs.docker.com/network/iptables/#restrict-connections-to-the-docker-host).

>  Would it not be better if we move the list blocked_interfaces to the globals section?

I thought it might be nice to have options that don't correlate directly with a [dockerd config entry](https://docs.docker.com/engine/reference/commandline/dockerd/) in a separate section.
I chose `firewall` because these settings modify the firewall for `dockerd`.

> would it not be better if we did not block everything and only allow those who are on the list.

I didn't consider this option thinking it should relate to the docs, but I think it would be better. We'd just have to enumerate all interfaces and add drops for all not mentioned, I'll look into this.

Do you think only the `lan` should be allowed by default? maybe not the OpenWRT interfaces and just the loopback interface and people would have to look up what `ethX` they wanted to allow?